src/routeConfiguration.js

```diff
@@ -108,34 +108,39 @@ const routeConfiguration = () => {
       component: () => (
         <NamedRedirect
           name="EditListingPage"
-          params={{ slug: draftSlug, id: draftId, type: 'new', tab: 'description' }}
+          params={{ slug: draftSlug, id: draftId, listingType: 'sauna', type: 'new', tab: config.firstSaunaTab }}
         />
       ),
     },
+
     {
       path: '/l-equipment/new',
       name: 'NewEquipmentListingPage',
       auth: true,
       component: () => (
         <NamedRedirect
           name="EditEquipmentListingPage"
-          params={{ slug: draftSlug, id: draftId, type: 'new', tab: 'general' }}
+          params={{ slug: draftSlug, id: draftId, listingType: 'equipment', type: 'new', tab: config.firstEquipmentTab }}
         />
       ),
     },
     {
       path: '/l/:slug/:id/:type/:tab',
       name: 'EditListingPage',
       auth: true,
-      component: EditListingPage,
+      component: (props) => (
+        <EditListingPage listingType='sauna' {...props} />
+      ),
       loadData: pageDataLoadingAPI.EditListingPage.loadData,
     },
     {
       path: '/l-equipment/:slug/:id/:type/:tab',
```

Comment:
forgot to change this 'l-equipment' to just 'l'. this has been fixed by the very next commit